BrainPortal/public/javascripts/cbrain.js

```diff
@@ -243,6 +243,18 @@
     });
 
 
+    loaded_element.find(".hide_qc_panel").click(function(event) {
```

Comment:
Please add a comment section explaining what the new code does; no need to go into details but indicate clearly what part of the user interface is affected an how.